### PR TITLE
fix: modify result for add-default-resources

### PR DESCRIPTION
**Pull request by MariamFahmy98**
## Related Issue(s)
https://github.com/kyverno/kyverno/issues/6975

## Description
use `skip` as a result in `kyverno-test.yaml` as mutation won't occur on `nginx-demo2` because it has both memory and cpu fields.

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

File: other/add-default-resources/kyverno-test.yaml [hunk 1]
```diff
@@ -10,13 +10,12 @@ results:
     patchedResource: patchedResource1.yaml
     kind: Pod
     result: pass
-# nginx-demo2 will get skip as resource already has memory and cpu requests
   - policy: add-default-resources
     rule: add-default-requests
     resource: nginx-demo2
     patchedResource: patchedResource2.yaml
```
Comment (chipzoller) [inline other/add-default-resources/kyverno-test.yaml:16]: You may need to remove this.

**Review by chipzoller — COMMENTED** [review]
Test currently fails for this.

File: other/add-default-resources/kyverno-test.yaml (hunk 1 above)
Comment (MariamFahmy98) [reply other/add-default-resources/kyverno-test.yaml]: done.

Comment (chipzoller) [reply other/add-default-resources/kyverno-test.yaml]: Still not succeeding. When you test locally, what is the result? It has to be either `skip` or `pass`.

**Comment by eddycharly** [comment]
I think the test fails because we have a cyclic dependency between this repo and the kyverno repo

**Comment by chipzoller** [comment]
Yes, modifications made here require modifications already be made in kyverno/kyverno in `main`.

**Comment by eddycharly** [comment]
May be easier to merge this first, kyverno repo doesn't allow merging PRs if tests fail.

**Comment by chipzoller** [comment]
Ok

File: other/add-default-resources/kyverno-test.yaml (hunk 1 above)
Comment (MariamFahmy98) [reply other/add-default-resources/kyverno-test.yaml]: Here's the result I got when it's `skip`:
```
Executing add-default-resources...
applying 1 policy to 3 resources... 

skipped mutate policy add-default-resources -> resource default/Pod/nginx-demo2
│────│───────────────────────│──────────────────────│──────────────────│────────│
│ ID │ POLICY                │ RULE                 │ RESOURCE         │ RESULT │
│────│───────────────────────│──────────────────────│──────────────────│────────│
│  1 │ add-default-resources │ add-default-requests │ /Pod/nginx-demo1 │ Pass   │
│  2 │ add-default-resources │ add-default-requests │ /Pod/nginx-demo2 │ Pass   │
│  3 │ add-default-resources │ add-default-requests │ /Pod/nginx-demo3 │ Pass   │
│────│───────────────────────│──────────────────────│──────────────────│────────│

Test Summary: 3 tests passed and 0 tests failed
```

**Pull request merged**